javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/SymbolSolverWithJavassistClassTest.java

```diff
@@ -83,10 +84,10 @@ void testSolveSymbolInTypeCantResolveNonExistentField() {
         assertFalse(solvedSymbol.isSolved());
 
         try {
-            solvedSymbol.getCorrespondingDeclaration();
+            solvedSymbol.getCorrespondingDeclaration().get();
         } catch (Exception e) {
-            assertTrue(e instanceof UnsupportedOperationException);
-            assertEquals("CorrespondingDeclaration not available for unsolved symbol.", e.getMessage());
+            assertTrue(e instanceof NoSuchElementException);
+            assertEquals("No value present", e.getMessage());
             return;
         }
```

Comment:
This might be unavoidable (I've not gotten to that bit yet) but on the face of it this error message seems like it is (potentially) more vague -- stating that there's no value present, as opposed to directing the reader to a likely cause of the symbol being unsolved.

Can we make this more descriptive - either within this PR or as a follow up?

Reply:
I agree with you that the previous message was more descriptive than this one. But i don't think it would be possible to change that message, since the `NoSuchElementException` is create inside of `Optional#get()` method with an hard coded string :/